package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "fss-node-metainspector",
   "description": "Npm package for web scraping purposes. You give it an URL, and it lets you easily get its title, links, images, description, keywords, meta tags",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "homepage": "http://github.com/gabceb/node-metainspector",
   "repository": "git://github.com/gabceb/node-metainspector.git",
+  "publishConfig": {
```

Comment:
Since `publishConfig` is specific to our Follett fork and I'd like to submit your fix to the original repo, let's put this change in a separate PR against **patched**.